src/game-support/lba-twinsens-quest.md

```diff
@@ -0,0 +1,20 @@
+# Little Big Adventure – Twinsen’s Quest
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Bronze installs=Yes opens=Yes}}
+
+Game crashes when run without modification.
+
+# Technical background
+
+A patch is required to make the game run. `Rewired_WindowsGamingInput.dll` found under the data directory in `Plugins/x86_64` makes the game run, although some issues surrounding use of controllers remain. 
+
+The function `Core_IsAPISupported` in the aforementioned DLL needs to be patched to always return false. This prevents calling an API that isn't implemented.
+
+# Patching
+
+For the DLL with SHA256 `946478c5e8f61cbe75c6d70e3aabe2766126605cba45ddb579a5ef38fb198e98` three bytes at offset 0x600 need to be changed to `b0 00 c3`. The resulting file should have a SHA256 of `954fa6b2960b2567797f061ec11131d026c1c640e5dfb8e9d4d2bae593151d36`
+
+# Issues
+
+Game is playable through the end, with all cutscenes playing correctly and no graphical corruption observed. Playing with a controller sometimes makes Twinsen perform an action continuously. This issue only occurs when loading or starting a game and is resolved by quitting and restarting the game.
```

Comment:
Please add a Steam game template with an ID of 2318070 at the cery bottom of the file. You can find examples in other titles there. It helps users to locate the game on Steam.

Reply:
I'll add it, though I didn't acquire the game through Steam and given the binary patching required wasn't willing to guess the GOG version exactly matched the Steam one.

Reply:
Sure. I will have to think about handling such cases as this Steam ID requirement is convenient from front-end side but it is not clear wheter to include it or not for non-Steam-tested games. Currently I decided to add it to keep it consistent with other titles. Thanks for your understanding and pushes.